src/mol-repr/structure/complex-visual.ts

```diff
@@ -157,9 +161,13 @@ export function ComplexVisual<G extends Geometry, P extends StructureParams & Ge
                 throw new Error('expected renderObject to be available');
             }
 
+            if (updateState.updateColor || updateState.updateSize || updateState.updateTransform) {
+                // console.log('update locationIterator');
```

Comment:
```suggestion
```

And the others too?

Reply:
I often use those log statements so I like to keep them around (commented out since we don't have good way for verbose logging).